Hardware/Nic/NicGroup.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.NetworkInformation;
+using System.Text;
+
+namespace OpenHardwareMonitor.Hardware.Nic
+{
+    internal class NicGroup : IGroup
+    {
+
+        private List<Hardware> hardware = new List<Hardware>();
+        private NetworkInterface[] nicArr;
+
+        public NicGroup(ISettings settings)
+        {
+            int p = (int)Environment.OSVersion.Platform;
+            if ((p == 4) || (p == 128))
+            {
+                hardware = new List<Hardware>();
+                return;
+            }
+            nicArr = NetworkInterface.GetAllNetworkInterfaces();
+            for (int i = 0; i < nicArr.Length; i++)
+            {
+                if (nicArr[i].NetworkInterfaceType.ToString() == "Ethernet")
+                {
+                    hardware.Add(new Nic(nicArr[i].Name, settings, i, this));
+                }
+                
+            }
+                
+        }
+
+        public string GetReport()
+        {
+            return null;
```

Comment:
I think it would be useful to add **some** print here for reporting, should be rather trivial to add a short summary of network interfaces and traffic.

Reply:
Yes  -> It's on Todo list :-)

Reply:
I'll add some here, working on it now.